.github/workflows/ci.yml

```diff
@@ -0,0 +1,197 @@
+            type=raw,value=latest,enable=${{ !contains(steps.ver.outputs.tag, 'rc') }}
+            type=raw,value=latest-focal,enable=${{ !contains(steps.ver.outputs.tag, 'rc') }}
+            type=raw,value=${{ steps.ver.outputs.tag }}
+            type=raw,value=${{ steps.ver.outputs.tag }}-focal
+          flavor: |
+            latest=false
+      - name: Docker metadata distroless
+        id: distroless-meta
+        uses: docker/metadata-action@v3
+        with:
+          images: ${{ steps.packageName.outputs.package_name }}
+          tags: |
+            type=raw,value=latest-distroless,enable=${{ !contains(steps.ver.outputs.tag, 'rc') }}
+            type=raw,value=${{ steps.ver.outputs.tag }}-distroless
+          flavor: |
+            latest=false
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Push image
+        uses: docker/build-push-action@v2
+        with:
+          context: ${{ steps.baseDirectory.outputs.directory }}/target/docker/stage
+          file: ${{ steps.baseDirectory.outputs.directory }}/target/docker/stage/Dockerfile
+          platforms: linux/amd64,linux/arm64/v8
+          tags: ${{ steps.meta.outputs.tags }}
+          push: true
+      - name: Push distroless image
+        uses: docker/build-push-action@v2
+        with:
+          context: ${{ steps.baseDirectoryDistroless.outputs.directory }}/target/docker/stage
+          file: ${{ steps.baseDirectoryDistroless.outputs.directory }}/target/docker/stage/Dockerfile
+          platforms: linux/amd64,linux/arm64/v8
+          tags: ${{ steps.distroless-meta.outputs.tags }}
+          push: true
+
+  create_release:
+    needs: test
+    if: startsWith(github.ref, 'refs/tags/')
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Github
+      uses: actions/checkout@v2
+    - uses: coursier/cache-action@v6
+    - name: Set up JDK 1.8 for batch transformer
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Set up JDK 11 for loader and streaming transformer
+      uses: actions/setup-java@v1
+      with:
+        java-version: 11
+    - name: Build redshift-loader jar
+      run: |
+        export SKIP_TEST=true
+        sbt 'project redshiftLoader' assembly
+    - name: Build snowflake-loader jar
+      run: |
+        export SKIP_TEST=true
+        sbt 'project snowflakeLoader' assembly
+    - name: Build transformer-batch jar
+      run: |
+        export SKIP_TEST=true
+        sbt 'project transformerBatch' assembly
+    - name: Build transformer-kinesis jar
+      run: |
+        export SKIP_TEST=true
+        sbt 'project transformerKinesis' assembly
+    - name: Get current version
+      id: ver
+      run: |
+        export PROJECT_VERSION=$(sbt "project common" version -Dsbt.log.noformat=true | perl -ne 'print "$1\n" if /info.*(\d+\.\d+\.\d+[^\r\n]*)/' | tail -n 1 | tr -d '\n')
```

Comment:
I prefer [this way of getting the current version](https://github.com/snowplow/enrich/blob/8b55ec27c6e31895141f17e2cadf39ef87f75226/.github/workflows/ci.yml#L137).

Now that we use sbt-dynver plugin, it is guaranteed that sbt and github "agree" on what is the current version.